srcpkgs/python-pykka/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'python-pykka'
 pkgname=python-pykka
-version=1.2.1
-revision=2
+version=2.0.2
+revision=1
 archs=noarch
 wrksrc="Pykka-${version}"
-build_style=python2-module
+build_style=python-module
 pycompile_module="pykka"
-hostmakedepends="python-setuptools"
+hostmakedepends="python-setuptools python3-setuptools"
 depends="python"
 short_desc="Python implementation of the actor model"
 maintainer="allan <mail@may.mooo.com>"
 homepage="https://pypi.org/project/Pykka/"
 license="Apache-2.0"
 distfiles="${PYPI_SITE}/P/Pykka/Pykka-${version}.tar.gz"
-checksum=e847ffeadee49b563426ab803e8eee67264d773e38ca14763fdcda56411e3c11
+checksum=895cc2ed8779b65dd14a90ba3f4b8cb0f7904c7bf0710fe96a923019f8e82a39
```

Comment:
You are missing
python3-pykka_package() 
.....
and the ln -s srcpkgs/python-pykka srcpkgs/python3-pykka

Reply:
Oh ok ! I'm on it

Reply:
I missed the trick. My bad!